incubator/command-dump/pom.xml

```diff
@@ -91,8 +103,8 @@
         <artifactId>license-maven-plugin</artifactId>
         <configuration>
           <excludes>
-            <exclude>src/test/resources/io/aklivity/zilla/runtime/command/dump/internal/*</exclude>
-            <exclude>src/test/resources/io/aklivity/zilla/runtime/command/dump/internal/engine/*</exclude>
+            <exclude>src/test/resources/io/aklivity/zilla/runtime/command/dump/internal/airline/*</exclude>
+            <exclude>src/test/resources/io/aklivity/zilla/runtime/command/dump/internal/airline/engine/*</exclude>
```

Comment:
Can we scope the files a little more tightly so that new resources get caught automatically?
For example `src/test/resources/**/*.pcap` etc.

Reply:
fixed, pls chk